packages/vuepress-theme-odyssey/styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,15 @@
+$breakpoints: (
+  m: 768px,
+  l: 1240px,
+  lg: 1800px
```

Comment:
We had things like desktop, tablet, mobile, etc and that was slighly helpful to go with, without having to come back and read what the breakpoint limits were.

Reply:
@santhoshbalakrishnan we're trying to stay device agnostic so far - these are probably going to get another update as @joejezowski-okta finalizes the best breakpoints for our new layout.

Reply:
This is a good call out, we explicitly decided not to target device widths and instead focus on available space. The breakpoint values chosen don't make it seem that way - but they will likely change!